Codeinwp/optimole-wp
# Follow the WP guidelines and use greyscale version of the logo in wp-admin + minify SVG

### All Submissions:

 

### Changes proposed in this Pull Request:
 
First of all, I love your plugin, thanks for the great work. However, the icon stands out too much from the other plugins without any particular reason.

This PR makes a change from this:

![SCR-20240514-oqqw](https://github.com/Codeinwp/optimole-wp/assets/1534150/fa8117a7-709d-405b-886e-cc3223bc50bc)

To this:

![SCR-20240514-ouvf](https://github.com/Codeinwp/optimole-wp/assets/1534150/78f9ad20-8e04-4900-9a99-a500f31d80fa)

It follows the same logo that is on your website without the circle bg:

![SCR-20240514-ovdd](https://github.com/Codeinwp/optimole-wp/assets/1534150/a67eb8ee-a52f-47a3-9799-ed8c3a6f2cbd)

This PR also minifies the SVG.

### How to test the changes in this Pull Request:

1. Get the latest version with this change
2. See the effect immediately

I hope you consider this change as you already have a paid service and that logo does not need to stand out there in the UI all the time you look at your WordPress site. Thanks in advance!

### Other information:
